client/src/components/Form/Elements/parameters.js

```diff
@@ -189,22 +185,6 @@ export default Backbone.View.extend({
         });
     },
 
-    /** Hidden field */
-    _fieldHidden: function (input_def) {
-        return new Ui.Hidden({
-            id: `field-${input_def.id}`,
-            info: input_def.info,
-        });
-    },
-
-    /** Boolean field */
-    _fieldBoolean: function (input_def) {
-        return new Ui.Switch({
```

Comment:
Not related to this PR. But I just wanted to ask, is Ui.Switch used anywhere else? I couldn't find the usage, is it's really the case, why don't we delete entire `ui-switch.js` then?

Reply:
ui-switch is used `NullableText` input element in `ui-misc.js`.

Reply:
ah, you mean https://github.com/galaxyproject/galaxy/blob/fd08c0a2e087450a45b5f80bb8f8fc2c55495568/client/src/mvc/ui/ui-misc.js#L137? yeah, that's right, sorry :+1: